tailwind.config.js

```diff
@@ -2,7 +2,24 @@
 module.exports = {
   content: ["./src/**/*.{html,js}"],
   theme: {
-    extend: {},
+    extend: {
+      fontFamily: {},
```

Comment:
you may need to put your fonts here? I don't remember how tailwind and vanilla css files work together